.github/workflows/validate-release.yml

```diff
@@ -19,53 +19,21 @@ on:
   push:
     branches:
       - main
-      - 'release-**'
+      - release-*
```

Comment:
I had quoted these since they were quoted in https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-branches-and-tags

Reply:
done